docs/Partitioner.md

```diff
@@ -0,0 +1,52 @@
+## Graph Partitioning Support in Glow
+
+### Overview
+Along with the increased size of neural networks, a large network may not be able to run on only one accelerator/device due to resource limitation (e.g. memory capacity). Therefore, it is necessary to divide up the network into several sub-networks and each of them can be executed on one accelerator. This problem can be abstracted as a Graph Partition problem [https://en.wikipedia.org/wiki/Graph_partition] : Given a neural network and n devices, we would like to split this network into minimal k sub-networks (where k <= n) , and for each sub-network, it can be run on one device with the device resource restriction.
```

Comment:
nit: (Graph Partition problem)[https://en.wikipedia.org/wiki/Graph_partition]
this looks a bit nicer.